nb_conda_kernels/runner.py

```diff
@@ -63,7 +63,19 @@ def exec_in_env(conda_root, envname, command, *args):
 
     # Launch the kernel process
     if is_win:
-        Popen((fullpath,) + args, env=env).wait()
```

Comment:
@jankatins I know you've been involved in the discussion about the R process not dying when a kernel is terminated. This ended up causing difficulties with my new, notebook-free testing approach for this package, because AppVeyor won't quit (and will time out with an error) if there are stray processes left by the build job.

I've implemented a windows job group here that takes care of the problem, inspired by the StackOverflow answer provided in the comments. I'm wondering whether or not I should bother, since it's technically not my problem to deal with this here, but rather in jupyter_client or IRkernel or...

Reply:
Thats https://github.com/jupyter/jupyter_client/issues/104 

IMO the fix on the jupyter_client side would be quite trivial (using psutil), I just never got to it...

Reply:
I experimented with psutil, but I had some issues with it. I like the automatic and comprehensive result here.

Reply:
Can you elaborate on that problem with psutil? Would be interesting to take that into account if a solutions will be tried in jupyter client.

Reply:
So I was using `psutil` in `test_runner.py` to collect the PID's of the runaway child processes. My thinking was that I shouldn't bother to do anything to try and clean them up in the functional code, but I need to do something in the tests. But the problem was that there was a stray R process left even after I terminated then killed all of the children `psutil` listed for me. So somehow, the R process was being treated as if it _wasn't_ a child. I don't fully  understand why, but enough failed AppVeyor jobs convinced me I needed to go another route.